pkg/render/common/securitycontext/security_context.go

```diff
@@ -76,3 +76,8 @@ func NewNonRootPodContext() *corev1.PodSecurityContext {
 		},
 	}
 }
+
+// GetNonRootUID returns the non-root UID
```

Comment:
You can revert this change and use `10001` directly in your test. We do this in other places as well, for example, https://github.com/tigera/operator/blob/88d6475f1f63e0249fc6acdd10bd52c018951bf4/pkg/render/node_test.go#L881.